kahuna/public/js/preview/image.html

```diff
@@ -46,6 +46,13 @@
 
     <div class="preview__info" ng:if="! ctrl.hideInfo">
         <div class="flex-container">
+            <span class="preview__info__collection"
+                ng-repeat="collection in ctrl.image.data.collections"
+                gr:tooltip="{{collection.data.path.join(' ▸ ')}}">
```

Comment:
We could actually create the template in the controller and just pass it through?